### DISCUSSION TITLE
there is a problem when Successfully build a cluster

### DISCUSSION THREAD

**Discussion by moziie** [emqx/emqx#5037] [Q&A] [answered]
After I set up the cluster, there are two nodes. How should I configure it to automatically connect to the other node when one of the nodes goes down?
At present, my program is connected to one of the nodes, and when this node goes down, it loses the connection and fails to achieve the purpose of the cluster.
I need help,please.

**Comment by tigercl** [comment, marked as answer]
@moziie Sorry for late reply. Usually we recommend that you use HAProxy or Nginx to load balance connections.
